src/models/devicemodel.h

```diff
@@ -42,7 +42,7 @@ class DeviceModel final : public QAbstractListModel {
 
   void removeDeviceFromPublicKey(const QString& publicKey);
 
-  int activeDevices() const { return m_devices.count(); }
+  int activeDevices() const { return static_cast<int>(m_devices.count()); }
```

Comment:
n00b question: Why do we need the static_cast here? Weren't we returning an `int` before already?

Reply:
count() returns a `qsizetype`, which means a `size_t`which is a `unsigned int`.

Reply:
Ok, but the function signature was not changed, which would mean it was already returning an `int` before. What that some implicit type coersion? What is different about that and the static_cast?

Reply:
Nevermind, this was probably to address warnings. Right?

Reply:
Right!